language_names/README.md

```diff
@@ -11,9 +11,9 @@
 
 ### Manual - EDIT
 
-- `manual.sql` contains 'fixes' and 'additions' to the scraped data.
-- `variants.sql` contains additions to the scraped data for language variants.
-- `turkic.sql` contains Turkic language names.
+- `manual.tsv` contains 'fixes' and 'additions' to the scraped data.
```

Comment:
This needs to be updated for the new files.

Reply:
Ohh forgot about it